src/tests/snapshots/gitu__tests__push__open_push_menu_after_dash_input.snap

```diff
@@ -18,8 +18,8 @@ expression: ctx.redact_buffer()
                                                                                 |
 ────────────────────────────────────────────────────────────────────────────────|
 Push                    Arguments                                               |
-p to default            -n Dry run (--dry-run)                                  |
-e to elsewhere          -F Force (--force)                                      |
-q/<esc> Quit/Close      -f Force with lease (--force-with-lease)                |
-                        -h Disable hooks (--no-verify)                          |
-styles_hash: 5f1105080c8b8328
+p to upstream           -n Dry run (--dry-run)                                  |
+r to pushRemote         -F Force (--force)                                      |
```

Comment:
Would be sweet if the remote/branch was visible here. I'm afraid users will be bamboozled by the change of behavior.

Seems like a bit of extra work though :thinking:.
Could be another issue/PR.